pages/doc/logging_overview.md

```diff
@@ -3,16 +3,16 @@ title: Get Started with Logs (Beta)
 tags: [getting started, logs]
 sidebar: doc_sidebar
 permalink: logging_overview.html
-summary: Learn about Tanzu Observability metrics, logs, and traces.
+summary: Learn about VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront) metrics, logs, and traces.
 ---
 
-{% include important.html content="Tanzu Observability Logs (Beta) is enabled only for selected customers. To participate, contact your Tanzu Observability account representative or [technical support](wavefront_support_feedback.html#support)."%}
+{% include important.html content="Logs (Beta) is enabled only for selected customers. To participate, contact your account representative or [technical support](wavefront_support_feedback.html#support)."%}
 
  <table style="width: 100%;">
 <tbody>
 <tr>
   <td width="60%" >
-    Tanzu Observability helps you monitor your application using metrics, traces, and logs.
+    VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront) helps you monitor your application using metrics, traces, and logs.
```

Comment:
You already have "(formerly known as Tanzu Observability by Wavefront)" on top, so I would leave only the new name here.

VMware Aria Operations for Applications helps you monitor your application using metrics, traces, and logs.